spec/integration/data_stream/manifest.spec.yml

```diff
@@ -338,17 +338,21 @@ spec:
                         type: object
                         additionalProperties: false
                         properties:
-                          type:
-                             $ref: "./fields/fields.spec.yml#/items/properties/type"
-                          scaling_factor:
-                            $ref: "./fields/fields.spec.yml#/items/properties/scaling_factor"
-                          ignore_malformed:
-                            $ref: "./fields/fields.spec.yml#/items/properties/ignore_malformed"
+                          doc_values:
+                            $ref: "./fields/fields.spec.yml#/items/properties/doc_values"
```

Comment:
According to the JSON patches, this field (and `index`) will be available since package spec 3.0.0. It looks ok to me.

https://github.com/elastic/package-spec/blob/6a114899b06b8a1aaa87832439fb853652b4a9e9/spec/integration/data_stream/manifest.spec.yml#L523